kakarot_scripts/deployment/kakarot_deployment.py

```diff
@@ -68,6 +69,12 @@ async def deploy_or_upgrade_kakarot(owner):
             DEFAULT_GAS_PRICE,
             address=starknet_deployments["kakarot"],
         )
+        await invoke(
```

Comment:
so coinbase is the default EVM address set in the .env ?

Reply:
initially, so that the EVM deployer pays for itself, until coinbase is deployed

Reply:
added more context about this